controllers/lookup/k8s_client.go

```diff
@@ -0,0 +1,160 @@
+package lookup
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/datainfrahq/druid-operator/apis/druid/v1alpha1"
+	internalhttp "github.com/datainfrahq/druid-operator/pkg/http"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"net/http"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type K8sClient struct {
```

Comment:
This is not needed.
The following functions (e.g. FindLookups) are not functions of `K8sClient` but are functions of `DruidLookupReconciler`.

Remove this struct and change functions ownership

Reply:
Removed `K8sClient` and moved associated methods to `DruidLookupReconciler` in c6b23dd.